### PR TITLE
Save predictions

### PR DESCRIPTION
**Pull request by dbolson**

### CODE REVIEW

File: application.py [hunk 1]
```diff
@@ -26,9 +26,7 @@ def predictions(district_number):
     prediction = Prediction(db).find(dict(
         district_number=district_number,
         dollars_spent=request.form.get('dollars_spent', None),
-        election_type=request.form.get('election_type'),
```
Comment (dbolson) [inline application.py:29]: I don't see `election_type` and `is_encumbent` used, based on the predictions csv, so I got rid of these.

Comment (mdeland) [reply application.py:29]: in that case could add `turnout` ?

Comment (dbolson) [reply application.py:29]: We're using `turnout` below?

Comment (mdeland) [reply application.py:29]: whoops, sorry

**Pull request merged**